test/parallel/test-event-emitter-remove-all-listeners.js

```diff
@@ -38,24 +38,25 @@ function expect(expected) {
 
 {
   const ee = new events.EventEmitter();
-  ee.on('foo', common.noop);
-  ee.on('bar', common.noop);
-  ee.on('baz', common.noop);
-  ee.on('baz', common.noop);
+  const noop = common.mustNotCall();
```

Comment:
Happy to do that if there's consensus that it would be an improvement. I didn't do it because:

* It's not needed as an identifier in the other scopes. (It's needed here because of the `assert.deepStrictEqual()` comparisons.)

* Fewer global identifiers shared across different block scopes helps make those things easily cut-and-paste-able into separate tests for refactoring, etc. It also helps limit the possibility of side effects. (For example, if another test attaches a property to the `noop` function for some reason, that property will be set for all the other tests.)

But this isn't far off from a tabs-vs-spaces discussion, so I'm happy to just do it if everyone else thinks differently than me. :-D